tests/unit/instance-initializers/router-test.ts

```diff
@@ -0,0 +1,32 @@
+import Application from '@ember/application';
+
+import { initialize } from 'ember-osf-web/instance-initializers/router';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import destroyApp from '../../helpers/destroy-app';
+
+module('Unit | Instance Initializer | router', function(hooks) {
+    setupTest(hooks);
+
+    // TODO: Actually test
+    hooks.beforeEach(function() {
+        this.TestApplication = Application.extend();
```

Comment:
This is not the proper way to do this anymore. See: https://discord.com/channels/480462759797063690/1017095930119454820